src/generators/cfml.ts

```diff
@@ -0,0 +1,116 @@
+import * as util from "../util.js";
+import type { Request } from "../util.js";
+
+import jsesc from "jsesc";
+
+const quote = (str: string): string => {
+  return jsesc(str, { quotes: "single" }).replace(/"/g, '""');
+};
+
+export const _toCFML = (request: Request): string => {
+  let cfmlCode = "";
+
+  cfmlCode += "httpService = new http();\n";
+  cfmlCode += 'httpService.setUrl("' + quote(request.url as string) + '");\n';
+  cfmlCode += 'httpService.setMethod("' + quote(request.method) + '");\n';
+
+  if (request.cookies) {
+    for (const [headerName, headerValue] of request.cookies) {
+      cfmlCode +=
+        'httpService.addParam(type="cookie", name="' +
+        quote(headerName) +
+        '", value="' +
+        quote(headerValue) +
+        '");\n';
+    }
+    util.deleteHeader(request, "Cookie");
+  }
+
+  if (request.headers && request.headers.length) {
+    for (const [headerName, headerValue] of request.headers) {
+      cfmlCode +=
+        'httpService.addParam(type="header", name="' +
+        quote(headerName) +
+        '", value="' +
+        quote(headerValue as string) +
+        '");\n';
+    }
+  }
+
+  if (request.timeout) {
+    cfmlCode +=
+      'httpService.setTimeout(' + (parseInt(request.timeout) || 0) + ');\n';
+  }
+
+  if (request.proxy) {
+    let proxy = request.proxy
+    let proxyPort = '1080';
+    const proxyPart = (request.proxy as string).match(/:([0-9]+)/);
+    if (proxyPart) {
+      proxy = request.proxy.slice(0, proxyPart.index);
+      proxyPort = proxyPart[1]
+    }
+
+    cfmlCode += 'httpService.setProxyServer("' + quote(proxy) + '");\n';
+    cfmlCode += 'httpService.setProxyPort(' + quote(proxyPort) + ');\n';
+
+    if (request.proxyAuth) {
+      const [proxyUser, proxyPassword] = request.proxyAuth.split(/:(.*)/s, 2);
+      cfmlCode += 'httpService.setProxyUser("' + quote(proxyUser) + '");\n';
+      cfmlCode +=
+        'httpService.setProxyPassword("' + quote(proxyPassword || "") + '");\n';
+    }
+  }
+
+  if (request.auth) {
+    cfmlCode +=
+      'httpService.addParam(type="header", name="Authorization", value="' +
+      quote(request.auth.join(":")) +
+      '");\n';
+  }
+
+  if (request.data || request.multipartUploads) {
+    if (request.multipartUploads) {
+      for (const [multipartKey, multipartValue] of request.multipartUploads) {
+        if (multipartValue.charAt(0) === "@" && !request.isDataRaw) {
```

Comment:
sorry, this was a bad suggestion, it's not applicable to `-F`
```suggestion
        if (multipartValue.charAt(0) === "@") {
```